packages/flutter_test/lib/src/controller.dart

```diff
@@ -1962,7 +1965,7 @@ abstract class WidgetController {
   /// soft keyboard.
   ///
   /// Specify `platform` as one of the platforms allowed in
-  /// [platform.Platform.operatingSystem] to make the event appear to be from
+  /// [Platform.operatingSystem] to make the event appear to be from
```

Comment:
@srawlins for some reason dartdoc thinks these references are not resolvable - even though the comment_references lint says everything is fine with the docImport for dart:io.

```
dartdoc:stderr:   error: unresolved doc reference [Platform.operatingSystem]
dartdoc:stderr:     from flutter_test.WidgetController.sendKeyDownEvent: (file:///b/s/w/ir/x/w/flutter/packages/flutter_test/lib/src/controller.dart:2031:16)
dartdoc:stderr:   error: unresolved doc reference [Platform.operatingSystem]
dartdoc:stderr:     from flutter_test.WidgetController.sendKeyEvent: (file:///b/s/w/ir/x/w/flutter/packages/flutter_test/lib/src/controller.dart:1992:16)
dartdoc:stderr:   error: unresolved doc reference [Platform.operatingSystem]
dartdoc:stderr:     from flutter_test.WidgetController.sendKeyRepeatEvent: (file:///b/s/w/ir/x/w/flutter/packages/flutter_test/lib/src/controller.dart:2098:16)
dartdoc:stderr:   error: unresolved doc reference [Platform.operatingSystem]
dartdoc:stderr:     from flutter_test.WidgetController.sendKeyUpEvent: (file:///b/s/w/ir/x/w/flutter/packages/flutter_test/lib/src/controller.dart:2062:16)
```

Reply:
I will look...

Reply:
Looks like this problem is resolved. 🎉